### ISSUE TITLE
Monolog Issue, Won't Install

### ISSUE THREAD

**Issue by jhull** [ConvertKit/ConvertKitSDK-PHP#14]
Trying to install through composer and I'm getting this:

```
 Conclusion: remove monolog/monolog 2.0.2
    - Conclusion: don't install monolog/monolog 2.0.2
    - convertkit/convertkitapi dev-master requires monolog/monolog ^1.0
```

**Comment by jhull** [comment]
Is anyone maintaining this? Is my only option to downgrade my install of `monolog`? As per something like this: https://github.com/laravel/laravel/pull/5106

**Comment by tnorthcutt** [comment]
@jhull I'd welcome a PR to fix this.

**Comment by jhull** [comment]
OK...first time ever running a PR...let me know if I need to do anything else

**Comment by n7studios** [comment]
Addressing in https://github.com/ConvertKit/ConvertKitSDK-PHP/pull/28, to be released as version `1.0`, rather than a potentially breaking change into the unversioned `dev-master`. Thanks!
